src/ts/components/seekbar.ts

```diff
@@ -652,7 +652,7 @@ export class SeekBar extends Component<SeekBarConfig> {
     // and mouseup handlers to the whole document. A seek is triggered when the user lifts the mouse key.
     // A seek mouse gesture is thus basically a click with a long time frame between down and up events.
     seekBar.on('touchstart mousedown', (e: MouseEvent | TouchEvent) => {
-      let isTouchEvent = BrowserUtils.isTouchSupported && e instanceof TouchEvent;
+      let isTouchEvent = BrowserUtils.isTouchSupported && e.constructor.name === 'Touchscreen' && e instanceof TouchEvent;
```

Comment:
I'm afraid, I think this check for the constructor is not very predictable/safe and there's an easier way.

I'd propose to check for `window.TouchEvent` instead. And I'd put that in a helper function that we'd then use in both places 🙂

```ts
function isTouchEvent(e: UIEvent): e is TouchEvent {
  return window.TouchEvent && e instanceof TouchEvent;
};
```

Reply:
hi @jmsn unfortunately  the `window.TouchEvent` is also `undefined` on IE, I tried this as well.

Reply:
yes and that's exactly why checking `window.TouchEvent` before `e instanceof TouchEvent` will prevent us from running into the error 🙂

Reply:
thanks for the suggestion @jmsn I just made the changes.